geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/coprocessor/aggregators/HBaseBinAggregator.scala

```diff
@@ -9,24 +9,20 @@
 package org.locationtech.geomesa.hbase.coprocessor.aggregators
 import org.geotools.factory.Hints
 import org.locationtech.geomesa.features.ScalaSimpleFeature
-import org.locationtech.geomesa.filter.function.BinaryOutputEncoder
-import org.locationtech.geomesa.filter.function.BinaryOutputEncoder.BIN_ATTRIBUTE_INDEX
 import org.locationtech.geomesa.hbase.HBaseFeatureIndexType
 import org.locationtech.geomesa.hbase.coprocessor.GeoMesaCoprocessor
 import org.locationtech.geomesa.index.iterators.{BinAggregatingScan, ByteBufferResult}
+import org.locationtech.geomesa.utils.bin.BinaryOutputEncoder
 import org.locationtech.geomesa.utils.geotools.GeometryUtils
 import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 import org.opengis.filter.Filter
 
 class HBaseBinAggregator extends BinAggregatingScan with HBaseAggregator[ByteBufferResult]
 
 object HBaseBinAggregator {
-  def bytesToFeatures(bytes : Array[Byte]): SimpleFeature = {
-    val sf = new ScalaSimpleFeature("", BinaryOutputEncoder.BinEncodedSft)
-    sf.setAttribute(1, GeometryUtils.zeroPoint)
-    sf.setAttribute(BIN_ATTRIBUTE_INDEX, bytes)
-    sf
-  }
+
+  def bytesToFeatures(bytes : Array[Byte]): SimpleFeature =
+    new ScalaSimpleFeature("", BinaryOutputEncoder.BinEncodedSft, Array(bytes, GeometryUtils.zeroPoint))
```

Comment:
I did a brief check and didn't see any other places we could do this. the old code was hard-coded to set the geom to index 1, so seems like we would have still screwed things up if we change the BIN_SFT. really we should probably make a method in BinaryOutputEncoder so we don't have to construct the attributes anywhere else, but eh...